app/tests/unit/helpers/console_error_throw.js

```diff
@@ -12,17 +12,17 @@ if (!process.env.LISTENING_TO_UNHANDLED_REJECTION) {
 
 if (!process.env.ALLOW_CONSOLE) {
   const originalError = global.console.error
-  const originalWarn = global.console.warn
+  // const originalWarn = global.console.warn
   global.console.error = (...args) => {
     originalError(...args)
     fail(
       `console.error in tests are forbidden (run with ALLOW_CONSOLE=true to show the console.error without failing)\n${args}`
     )
   }
-  global.console.warn = (...args) => {
-    originalWarn(...args)
-    fail(
-      `console.warn in tests are forbidden (run with ALLOW_CONSOLE=true to show the console.warn without failing)\n${args}`
-    )
-  }
+  // global.console.warn = (...args) => {
+  //   originalWarn(...args)
+  //   fail(
+  //     `console.warn in tests are forbidden (run with ALLOW_CONSOLE=true to show the console.warn without failing)\n${args}`
+  //   )
+  // }
```

Comment:
:eyes:

Reply:
warning in polkadot

Reply:
Yeah, now I see why you did it. It is failing all over. Let's get this PR merged :woman_dancing: ! We can do the app thing later